util/testimonials2rst

```diff
@@ -346,4 +346,8 @@ NR == 1 { next }
 }
 
 HERE="$(cd "$(dirname "$0")"; pwd -P)"
-testimonials2rst "$@" | fmt -80 | tee "${HERE}"/../documentation/source/overview/testimonials.rst
+
+# To directly write on the RST file, uncomment the line below. Otherwise, copy/paste and manually organize the output
+# of the terminal.
+#testimonials2rst "$@" | fmt -80 | tee "${HERE}"/../documentation/source/overview/testimonials.rst
+testimonials2rst "$@" | fmt -80
```

Comment:
To explain why I did that: that file is under git, so you can run `git diff` immediately to review and `git commit` if you approve of the changes.

Reply:
> that file is under git, so you can run git diff immediately to review and git commit if you approve of the changes.

i understand the rationale, but i am not familiar enough with git and usage of this function, so it took me 5min to: 1) realize that the file was overwritten and 2) find the right command to undo the automatic change (git checkout FILENAME). And i know myself: in 1 month i will forgot this discussion and i will redo the same thing (overwrite file, find the correct git command, etc.).